src/utils/helpers.ts

```diff
@@ -6,6 +7,7 @@ import {
   SKILLS,
   ACTIVITIES,
 } from './constants';
+const ua = require('useragent-generator');
```

Comment:
Can you update this to use an ES6 import? Note that this will cause TypeScript to require a module definition. I got it to work locally by creating a `global.d.ts` file in `src` that looked like: 
```ts
declare module 'useragent-generator' {
  const firefox: {
    (options: number | string | { version: string; os: string }): string;
  };
}

```
Normally you could just `yarn add -D @types/useragent-generator` but unfortunately there are no definitions for this library.

Reply:
Sure, will do. I've usually seen foreign code `.d.ts` files in `@types` dir, not sure if there's a convention for that. For this lib it would be `@types/useragent-generator.d.ts` - is that OK?

Also, we can keep the bare minimum for TS to work, as in your example, but it's not too difficult to generate & enhance a `.d.ts` file for the entire library. The lib doesn't change too frequently, last commit is from 2018. For version `1.1.0` it'd look like this:

```ts
declare module 'useragent-generator' {
  /********************
   *  Google Chrome   *
   /*******************/
  export function chrome(opt: number | string | { version: string, os: string }): string;
  export namespace chrome {
    function androidPhone(opt: number | string | { version: string, androidVersion: string, device: string }): string;
    function androidTablet(opt: number | string | { version: string, androidVersion: string, device: string }): string;
    function androidWebview(opt: number | string | { androidVersion: string, chromeVersion: string, device: string })
      : string;
    function chromecast(opt: number | string | { version: string }): string;
    function iOS(opt: number | string | { iOSVersion: string, chromeVersion: string, device: string }): string;
  }
  export function chromium(opt: number | string | { version: string, os: string }): string;
  /***************
   *   Firefox   *
   /*************/
  export function firefox(opt: number | string | { version: string, os: string }): string;
  export namespace firefox {
    function androidPhone(opt: number | string | { version: string, androidVersion: string, device: string }): string;
    function androidTablet(opt: number | string | { version: string, androidVersion: string, device: string }): string;
    function iOS(opt: number | string | { iOSVersion: string, device: string }): string;
  }
  /**************
   *   Safari   *
   /************/
  export function safari(opt: number | string | { version: string, os: string }): string;
  export namespace safari {
    function iOS(opt: number | string | { iOSVersion: string, safariVersion: string, device: string }): string;
    function iOSWebview(opt: number | string | { iOSVersion: string, safariVersion: string, device: string }): string;
  }
  /***********************
   *  Internet Explorer  *
   /*********************/
  export function ie(opt: number | string | { version: string, os: string }): string;
  export namespace ie {
    function windowsPhone(opt: number | string | { version: string, device: string }): string;
  }
  /**********************
   *   Microsoft Edge   *
   /********************/
  export function edge(opt: number | string | { version: string, chromeVersion: string, os: string }): string;
  /************************
   *  Search Engine Bots  *
   /**********************/
  export function googleBot(opt?: number | string | { version: string }): string;
  export function bingBot(opt?: number | string | { version: string }): string;
  export function yahooBot(): string;
}
```

Which one do you think should be added?

Reply:
That `@types` convention probably comes from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped). If you want you can commit the full type definitions, up to you. It would be a bonus to make a PR for [useragent-generator](https://github.com/pastelsky/useragent-generator). If they're willing to publish it we could come back and update this repo with the latest version later. Thanks again for your help!

Reply:
I've tried to add types to [useragent-generator](https://github.com/pastelsky/useragent-generator), but I've struggled with correct setup of babel/webpack, and decided not to do it, as it would take me too much time. So I've committed the full `@types/useragent-generator.d.ts` file.